src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/ListDefinitionsAction.java

```diff
@@ -69,58 +69,55 @@ public void handle(Request request, Response response) {
-            .build();
+        return AlmSettingGithub.newBuilder().setKey(settingDto.getKey())
+                .setUrl(requireNonNull(settingDto.getUrl(), "URL cannot be null for GitHub ALM setting"))
+                .setAppId(requireNonNull(settingDto.getAppId(), "App ID cannot be null for GitHub ALM setting"))
+                .setPrivateKey(
+                        requireNonNull(settingDto.getPrivateKey(), "Private Key cannot be null for GitHub ALM setting"))
+                .setClientId(Optional.ofNullable(settingDto.getClientId()).orElse(""))
+                .setClientSecret(Optional.ofNullable(settingDto.getClientSecret()).orElse("")).build();
     }
 
     private static AlmSettingAzure toAzure(AlmSettingDto settingDto) {
-        return AlmSettingAzure.newBuilder()
-            .setKey(settingDto.getKey())
-            .setPersonalAccessToken(requireNonNull(settingDto.getPersonalAccessToken(), "Personal Access Token cannot be null for Azure ALM setting"))
-            .build();
+        return AlmSettingAzure.newBuilder().setKey(settingDto.getKey())
+                .setUrl(requireNonNull(settingDto.getUrl(), "URL cannot be null for Azure ALM setting"))
+                .setPersonalAccessToken(requireNonNull(settingDto.getPersonalAccessToken(),
+                        "Personal Access Token cannot be null for Azure ALM setting"))
+                .build();
     }
 
     private static AlmSettingBitbucket toBitbucket(AlmSettingDto settingDto) {
-        return AlmSettingBitbucket.newBuilder()
-            .setKey(settingDto.getKey())
-            .setUrl(requireNonNull(settingDto.getUrl(), "URL cannot be null for Bitbucket ALM setting"))
-            .setPersonalAccessToken(requireNonNull(settingDto.getPersonalAccessToken(), "Personal Access Token cannot be null for Bitbucket ALM setting"))
-            .build();
+        return AlmSettingBitbucket.newBuilder().setKey(settingDto.getKey())
+                .setUrl(requireNonNull(settingDto.getUrl(), "URL cannot be null for Bitbucket ALM setting"))
+                .setPersonalAccessToken(requireNonNull(settingDto.getPersonalAccessToken(),
+                        "Personal Access Token cannot be null for Bitbucket ALM setting"))
+                .build();
     }
 
     private static AlmSettings.AlmSettingGitlab toGitlab(AlmSettingDto settingDto) {
-        AlmSettings.AlmSettingGitlab.Builder almSettingBuilder =  AlmSettings.AlmSettingGitlab.newBuilder()
-            .setKey(settingDto.getKey())
-            .setPersonalAccessToken(requireNonNull(settingDto.getPersonalAccessToken(), "Personal Access Token cannot be null for Gitlab ALM setting"));
+        AlmSettings.AlmSettingGitlab.Builder almSettingBuilder = AlmSettings.AlmSettingGitlab.newBuilder()
+                .setKey(settingDto.getKey()).setPersonalAccessToken(requireNonNull(settingDto.getPersonalAccessToken(),
+                        "Personal Access Token cannot be null for Gitlab ALM setting"));
```

Comment:
The changes in this file seem to just be code formatting, which makes it difficult to tell what has actually been changed. Could you reduce your change down to just lines with functional changes please?